docs/group_concepts/stages_concept.rst

```diff
@@ -0,0 +1,31 @@
+Launch Stages Overview
+============================
+
+PubSub+ Cloud organizes its products and features to belong to one of the following launch stages: General Availability, Beta, Early Access, Alpha, Deprecated. To gain access to features in certain launch stages, special requests may have to be made to support. For details on the meaning of each launch stage please see below. Note: There is no requirement that a feature should go through every launch stage.
+
+
+General Availability
+~~~~~~~~
+
+General availability products and features are open to all customers, covered by our SLAs, and are ready for production use.
+
+Beta
```

Comment:
Can you put the launch stages in the following order...
-  EA
-  Alpha
-  Beta
-  GA
-  Deprecated